documents/2-RTB/Documentazione_Interna/Norme_di_Progetto.typ

```diff
@@ -664,4 +671,87 @@ Al fine di promuovere un ambiente di lavoro efficace ed efficiente, che garantis
+
+== Affidabilità
+
+Questo parametro si concentra sulla valutazione della capacità del software di mantenere un certo livello di prestazioni nel tempo anche in presenza di malfunzionamenti.
+
+Le sotto-categorie di questo parametro sono:
+
+- _*Maturità*_: Capacità del software di evitare malfunzionamenti, guasti o errori in fase di esecuzione.
+- _*Tolleranza agli errori*_: Capacità del software di mantenere un certo livello di prestazioni anche in presenza di malfunzionamenti o errori.
+- _*Recuperabilità*_: Capacità del software di ripristinare un certo livello di prestazioni e di recuperare i dati in caso di malfunzionamenti.
+- _*Affidabilità delle informazioni*_: Precisione e affidabilità delle informazioni ritornate dal software.
+
+
+== Usabilità
+
+Questo è un aspetto fondamentale della qualità del software in quanto influisce direttamente sulla user experience durante l'interazione con il prodotto.
+
+Le sotto-categorie di questo parametro sono:
+
+- _*Comprensibilità*_: Capacità del software di essere compreso facilmente dall'utente.
+- _*Apprendibilità*_: Capacità del software di essere appreso facilmente dall'utente al primo utilizzo.
+- _*Operabilità*_: Facilità con cui un utente può portare a termine le operazioni desiderate in modo efficiente, senza incorrere in problemi e con un utilizzo normale di risorse.
+- _*Attrattività*_: Capacità dell'interfaccia e del design del software di essere piacevole e attraente per l'utente.
+- _*Aderenza all'usabilità*_: Capacità del software di aderire e rispettare nel suo ciclio di vita standard e convenzioni di usabilità.
+
+== Efficienza
+
+Si concentra sulla capacità del prodotto di avere un utilizzo delle risorse ottimale ed efficiente, oltre che di garantire prestazioni elevate e tempi rapidi di risposta.
+
+Le sotto-categorie di questo parametro sono:
+
+- _*Comportamento rispetto al tempo*_: Capacità del software di fornire risposte in tempi accettabili.
+- _*Utilizzo delle risorse*_: Capacità del software di utilizzare in modo efficiente le risorse a disposizione.
+- _*Aderenza all'efficienza*_: Capacità del software di aderire e rispettare nel suo ciclo di vita standard e convenzioni di efficienza.
+
+== Manutenibilità
+
+Questo parametro si concentra sulla capacità del software di essere modificato, corretto e migliorato nel tempo.
+
+Le sotto-categorie di questo parametro sono:
+
+- _*Analizzabilità*_: Capacità del software di essere analizzato facilmente per individuare errori e problemi o sezioni da migliorare.
+- _*Modificabilità*_: Capacità del software di essere modificato facilmente per correggere errori, migliorare le prestazioni e adattarsi a nuove esigenze senza generare nuovi errori.
+- _*Stabilità*_: Capacità del software di mantenere un certo livello di prestazioni anche in presenza di modifiche di evitare di inificiare il codice esistente durante le modifiche.
```

Comment:
743
-> _*Stabilità*_: Capacità del software di mantenere un certo livello di prestazioni anche in presenza di modifiche, evitando di compromettere il codice esistente.

forse così è più chiaro

Reply:
done